test/browsing.js

```diff
@@ -229,6 +229,107 @@ describe('Browsing', () => {
       ));
   }).timeout(20000);
 
+  describe('WebFetch partial content', () => {
+    it('fetch partial content', () => {
+      const content = `hello world, on ${Math.round(Math.random() * 100000)}`;
+      const numberOfBytes = content.length - 1;
+      const bytesToRetrieve = 6;
+      const startByte = 3;
+
+      return createRandomDomain(content, '/streaming.mp4')
+        .then((domain) => createAnonTestApp()
+          .then((app) => app.webFetch(`safe://${domain}/streaming.mp4`,
+                    { range: { start: startByte, end: startByte + bytesToRetrieve } })
+            .then((data) => {
+              should(data.body.toString()).equal('lo wor');
+              should(data.body.toString()).equal(
+                          content.substring(startByte, startByte + bytesToRetrieve));
+              should(data.headers['Content-Range']).equal(`bytes ${startByte}-${startByte + (bytesToRetrieve)}/${numberOfBytes}`);
```

Comment:
I think `numberOfBytes` here shouldn't be the last byte index but the total length